Revendor libnetwork @ 0025177e3d

espensuenson opened the pull request:
Issues that are fixed :
log the actual error when failing to add IPv6 route https://github.com/docker/libnetwork/pull/2444
Shorten controller ID in exec-root to not hit UNIX_PATH_MAX https://github.com/docker/libnetwork/pull/2443
Fixed getNetworkFromStore, which returned an incorrect struct https://github.com/docker/libnetwork/pull/2449
weighted scheduling methods constants for ipvs https://github.com/docker/libnetwork/pull/2453
Resolve "bridge fdb show" hang issue https://github.com/docker/libnetwork/pull/2456

Signed-off-by: Espen Suenson <mail@espensuenson.dk>

arkodg commented:
@espensuenson thanks for raising this PR, would appreciate if you could add some minimal documentation for the new commits in the PR description (https://github.com/docker/libnetwork/compare/96bcc0dae898308ed659c5095526788a602f4726...0025177e3dabbe0de151be0957dcaff149d43536)

selansen commented:
Also pls add below information as part of the PR.  This will help moby maintainers to understand  what is this new revendor PR brings to moby upstream.

Issues that are fixed : 
================
log the actual error when failing to add IPv6 route #2444
Shorten controller ID in exec-root to not hit UNIX_PATH_MAX #2443
Fixed getNetworkFromStore, which returned an incorrect struct #2449
weighted scheduling methods constants for ipvs #2453
Resolve "bridge fdb show" hang issue #2456

espensuenson committed:
Revendor libnetwork @ 0025177e3d

Issues that are fixed :
log the actual error when failing to add IPv6 route https://github.com/docker/libnetwork/pull/2444
Shorten controller ID in exec-root to not hit UNIX_PATH_MAX https://github.com/docker/libnetwork/pull/2443
Fixed getNetworkFromStore, which returned an incorrect struct https://github.com/docker/libnetwork/pull/2449
weighted scheduling methods constants for ipvs https://github.com/docker/libnetwork/pull/2453
Resolve "bridge fdb show" hang issue https://github.com/docker/libnetwork/pull/2456

Signed-off-by: Espen Suenson <mail@espensuenson.dk>

q-o-app commented:
good time
![IMG_20190910_133232_784](https://user-images.githubusercontent.com/55634822/66700517-d10fdb80-ecf9-11e9-9f5c-0ebd80ec6edb.jpg)

q-o-app commented:
![IMG_20190822_022925_858](https://user-images.githubusercontent.com/55634822/66700528-f13f9a80-ecf9-11e9-86da-9daf743b3207.jpg)

Rid commented:
Tests fail because we changed to use a short libnetwork ID in the exec-root in https://github.com/moby/moby/pull/39822, I have revendored libnetwork there, so this can be closed.
